<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoSessionProperty</class>
 <widget class="QDialog" name="QWoSessionProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>       Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>       Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="remoteLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>       Host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="host"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>   Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="portTip">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Default:22</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="loginNameLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>  LoginName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="loginName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="loginTypeLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>  LoginType:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="loginType"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="passwordLayout">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>   Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="identifyLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>   Identify:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="identify"/>
       </item>
       <item>
        <widget class="QToolButton" name="identifyBrowser">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../private/skins/black/pattern2.png</normaloff>../private/skins/black/pattern2.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="proxyLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>  ProxyJump:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="proxyJump"/>
       </item>
       <item>
        <widget class="QToolButton" name="proxyBrowser">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../private/skins/black/linkcfg.png</normaloff>../private/skins/black/linkcfg.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="commandLayout">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>   Commands:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="command"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="localLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>   BaudRate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudRate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>   DataBits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataBits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>     Parity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>   StopBits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopBits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>FlowControl:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="flowControl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>  GroupName:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnGroupAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../private/skins/black/add2.png</normaloff>../private/skins/black/add2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Memo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="memo"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnMore">
       <property name="text">
        <string>More</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
